configure

```diff
@@ -2028,7 +2027,7 @@ Optional Features:
   --enable-fparser-debugging
                           Build fparser with bytecode debugging functions
   --disable-cppunit       Build without cppunit C++ unit testing support
-  --disable-nanoflann     build without nanoflann KD-tree suppport
+  --disable-nanoflann     build without nanoflann KD-tree support
```

Comment:
So how many false positives did you end up with from scspell?  I can't imagine it has "nanoflann", "fparser", and "cppunit" in its dictionary.

Reply:
There were definitely *lots* of false positives. But scspell makes it pretty painless to update ~/.scspell/dictionary.txt, which is a human-readable text file with all your local words. So for common words like "libmesh" etc, you only have to add them to the dictionary once. Most of the false positives were triggered by our variable names in code, and those are much harder to ignore.